templates/ovs_stats.conf.j2

```diff
@@ -1,8 +1,16 @@
 LoadPlugin ovs_stats
 
 <Plugin ovs_stats>
-#  Port "6640"
-#  Address "127.0.0.1"
-#  Socket "/var/run/openvswitch/db.sock"
-#  Bridges "br0" "br_ext"
+{% if collectd_plugin_ovs_stats_port is defined %}
+  Port "{{ collectd_plugin_ovs_stats_port }}"
+{% endif %}
+{% if collectd_plugin_ovs_stats_address is defined %}
+  Address "{{ collectd_plugin_ovs_stats_address }}"
+{% endif %}
+{% if collectd_plugin_ovs_stats_socket is defined %}
+  Socket "{{ collectd_plugin_ovs_stats_socket }}"
+{% endif %}
+{% if collectd_plugin_ovs_stats_bridges is defined %}
+  Bridges {{ collectd_plugin_ovs_stats_bridges }}
```

Comment:
The bridges won't end up as strings here, no?

Reply:
They won't, ovs_stats didn't complain about the format, however, it also didn't initialise properly (I'm assuming) because there's no agent in the container